src/scss/responsive/_rating.scss

```diff
@@ -0,0 +1,78 @@
+@media only screen and (min-width: 768px) and (max-width: 1023px) {
+  .rating {
+    &__content {
+      padding: 100px 40px 110px 40px;
+    }
+    &__card {
+      padding: 50px 100px;
+      &-content {
+        margin-top: 60px;
+      }
+      &-describing {
+        font-size: 0.9em;
+      }
+      &-person {
+        justify-content: center;
+        margin-bottom: 20px;
+      }
+      &-person-img {
+        width: 100px;
+        height: 100px;
+      }
+      &-describing {
+        text-align: center;
+        padding: 0;
+        line-height: 2;
+      }
+    }
+    &__control {
+      margin-top: 40px;
+      &-directional {
+        display: none;
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 767px) {
```

Comment:
For breakpoint, it's can be better by define variable for them:
- $mobileMax;
- $tabletMin;
- $tabletMax;
- $desktopMin